schemas/test/en/test_new_routing_and_skipping_section_dependencies.json

```diff
@@ -0,0 +1,549 @@
+                        }
+                    ],
+                    "id": "primary-person-group",
+                    "title": "Primary Person"
+                },
+                {
+                    "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "reason-no-confirmation",
+                            "question": {
+                                "answers": [
+                                    {
+                                        "id": "reason-no-confirmation-answer",
+                                        "mandatory": false,
+                                        "options": [
+                                            {
+                                                "label": "I did not visit section 2, so confirmation was not needed",
+                                                "value": "I did not visit section 2, so confirmation was not needed"
+                                            },
+                                            {
+                                                "label": "I did, but it was removed from the path as I changed my answer to No on the skip question",
+                                                "value": "I did, but it was removed from the path as I changed my answer to No on the skip question"
+                                            }
+                                        ],
+                                        "type": "Radio"
+                                    }
+                                ],
+                                "id": "reason-no-confirmation-question",
+                                "title": "Why didn't you answer the age skipping confirmation question?",
+                                "type": "General"
+                            }
+                        }
+                    ],
+                    "id": "confirmation-group",
+                    "title": "Confirmation Question",
+                    "skip_conditions": {
+                        "when": {
+                            "or": [
+                                {
+                                    "==": [
+                                        {
+                                            "source": "answers",
+                                            "identifier": "skip-confirmation-answer"
+                                        },
+                                        "Yes"
+                                    ]
+                                },
+                                {
+                                    "==": [
+                                        {
+                                            "source": "answers",
+                                            "identifier": "skip-confirmation-answer"
+                                        },
+                                        "No"
+                                    ]
+                                }
```

Comment:
This might look a bit odd, but it is deliberate. I wanted to prove the answer was not on the path, even though it might have an answer in the store, we don't have an rule for that so this is the best I could do